java/ingest/ingest-runner/src/test/java/sleeper/ingest/IngestRecordsIT.java

```diff
@@ -78,18 +80,10 @@
 class IngestRecordsIT extends IngestRecordsTestBase {
     @Test
     void shouldWriteRecordsSplitByPartitionLongKey() throws Exception {
-        // Given
-        Range rootRange = new Range.RangeFactory(schema).createRange(field, Long.MIN_VALUE, null);
-        Region rootRegion = new Region(rootRange);
-        Partition rootPartition = createRootPartition(rootRegion, new LongType());
-        Range range1 = new Range.RangeFactory(schema).createRange(field, Long.MIN_VALUE, 2L);
-        Region region1 = new Region(range1);
-        Partition partition1 = createLeafPartition("partition1", region1, new LongType());
-        Range range2 = new Range.RangeFactory(schema).createRange(field, 2L, null);
-        Region region2 = new Region(range2);
-        Partition partition2 = createLeafPartition("partition2", region2, new LongType());
-        rootPartition = rootPartition.toBuilder().childPartitionIds(Arrays.asList(partition1.getId(), partition2.getId())).build();
-        StateStore stateStore = inMemoryStateStoreWithFixedPartitions(rootPartition, partition1, partition2);
+        List<Partition> partition = new PartitionsBuilder(schema)
+                .rootFirst("root")
+                .splitToNewChildren("root", "partition1", "partition2", 2L).buildList();
```

Comment:
It would be a bit clearer to name this variable `allPartitions` to show that it's a list. Same for the tests below

Reply:
If you inline it so that the list is passed straight to inMemoryStateStoreWithPartitions, and avoid assigning a variable at all, that would also make it clear that the partitions aren't used anywhere further down in the test.

If you use inMemoryStateStoreWithFixedPartitions, that would also show that the partitions aren't ever changed after this point.

This also applies to the other tests in this file.